### PR TITLE
addClientFeatures based from vishalxl

### PR DESCRIPTION
**Pull request by xrviv**

### CODE REVIEW

File: data/alphaAma.com.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  comment: Instance managed by AlphaAMA's author.
+tags:
+- client
+platforms:
+- web
+license: 
+progLang:
+- JS
+features:
+- kind1/post
```
Comment (Giszmo) [inline data/alphaAma.com.md]: social implies kind-1. No "/" in tags, please.

```suggestion
```

File: data/alphaAma.com.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+title: AlphaAMA.com
+web: https://alphaama.com/
+git: https://github.com/eskema/alphaama
+instances:
+- name: AlphaAMA
+  url: https://alphaama.com
+  comment: Instance managed by AlphaAMA's author.
+tags:
+- client
```
Comment (Giszmo) [inline data/alphaAma.com.md:10]:
```suggestion
tags:
- client
- social
```

File: data/alphaAma.com.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+platforms:
+- web
+license: 
+progLang:
+- JS
+features:
+- kind1/post
+- reply
+- mentions
+- reactions
```
Comment (Giszmo) [inline data/alphaAma.com.md]: These features are supported by most other social clients, too. Until we have the time to list all these details, we should not mark some as such. These are implied by "social".

File: data/astral.md [hunk 4]
```diff
@@ -15,11 +15,20 @@ tags:
...
+- web
 license: MIT
 progLang:
 - Vue
 - JS
+features:
+- kind1/post
+- reply
+- mentions
+- dms
```
Comment (Giszmo) [inline data/astral.md]: all features are implied by tags.

**Review by Giszmo — CHANGES_REQUESTED** [review]
This PR is introducing a lot of redundant attributes. We do not go into these details yet.

File: data/alphaAma.com.md (hunk 3 above)
Comment (xrviv) [reply data/alphaAma.com.md]: removing.

File: data/astral.md (hunk 4 above)
Comment (xrviv) [reply data/astral.md]: removing

**Review by Giszmo — APPROVED** [review]

**Pull request merged**